specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2017-07-01/resourcehealth.json

```diff
@@ -966,6 +966,10 @@
       "enum": [
         "default"
       ],
+      "x-ms-enum": {
```

Comment:
Can you give more details about this enum ? why this needs to be added

Reply:
I put the detailed context in the internal emails since it is long.